src/v/cloud_storage_clients/configuration.cc

```diff
@@ -33,6 +33,8 @@ build_tls_credentials(
     cred_builder.set_ciphersuites(
       {config::tlsv1_3_ciphersuites.data(),
        config::tlsv1_3_ciphersuites.size()});
+    cred_builder.set_minimum_tls_version(
```

Comment:
I'm intrigued, why is it set here, but not, for example, for the oidc client or the cloud_roles client?

Which also begs the question, should it be possible to configure this per client, or listener?

Reply:
> I'm intrigued, why is it set here, but not, for example, for the oidc client or the cloud_roles client?

Thanks for pointing that out, definitely a miss.  I'll make the changes, but it's probably not as big as a deal as ensuring our services can be selective of what TLS version they support.

> Which also begs the question, should it be possible to configure this per client, or listener?

Maybe in a future update.  For now, to get something done to help our customers was to provide a single cluster config.